.github/workflows/build.yml

```diff
@@ -4,128 +4,96 @@ name: 'Build'
 on: ['push', 'pull_request']
 
 jobs:
-  test:
-    name: 'Test'
+  # Build the RocksDB C library and cache the result.
+  librocksdb:
+    name: 'Build librocksdb'
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         os: [ubuntu-latest]
-          #py_ver: ['3.7', '3.8', '3.9']
-          #rocksdb_ver: ['v6.14.6', 'v6.11.4', 'v5.17.2']
-        py_ver: ['3.9']
-        rocksdb_ver: ['v6.14.6']
+        rocksdb_ver: ['v6.14.6', 'v6.11.4']
 
     steps:
-      - uses: actions/checkout@v2
-        name: 'Checkout source repository'
-
-      - uses: actions/setup-python@v2
-        name: 'Set up Python ${{ matrix.py_ver }}'
+      - uses: actions/cache@v2
+        id: cache-librocksdb
         with:
-          python-version: ${{ matrix.py_ver }}
+          key: ${{ matrix.os }}-librocksdb-${{ matrix.rocksdb_ver }}
```

Comment:
You would want `librocksdb-${{ matrix.os }}-${{ matrix.rocksdb_ver }` if the partial `restore-keys` are to have any effect. But...